### PR TITLE
Removed the storage folders related to kf and also eliminated the kf profile.

### PR DESCRIPTION
**Pull request by Azher2Ali**
**Complete Removal of KF Storage Components**
**Summary:**

With the full removal of Kids First (KF) project elements from the SCORe client, we have deleted the entire storage folder associated with KF. This cleanup eliminates unnecessary code without impacting the functionality of the application.

**Details:**

- [ ] **Directory Removed:**

The entire kf storage folder and its contents located at score-client/src/main/java/bio/overture/score/client/storage/kf have been deleted./

**- [ ] Service Relocation:**

The ScoreStorageService has been moved into the parent package (bio.overture.score.client.storage), aligning it with the AbstractStorageService.

**- [ ] Profile Removal:**

The KF profile has been removed from ScoreStorageService (#460).

**- [ ] Configuration Cleanup:**

The configuration value related to KFStorageService has been removed from StorageConfig. Specifically, the partSize configuration at lines 21-22 in StorageConfig.java has been deleted.

We have removed the partSize variable from StorageConfig.java, but we can see that it's been further used as an argument for:
 ```
@Bean
  public SimplePartCalculator partCalculator() {
    return new SimplePartCalculator(partSize);
```
`partCalculator()` is further being used in 

- AzureDownloadService

- AzureUploadService

- S3DownloadService

- S3UploadService

So removing the partsize variable from the StorageConfig.java will affect the following classes too.

**Suggested Changes:**

In order to resolve this issue, we need to pass a different argument in the return statement -- #458 & #460

### CODE REVIEW

File: score-client/src/main/java/bio/overture/score/client/config/StorageConfig.java [hunk 1]
```diff
@@ -18,9 +18,6 @@ public class StorageConfig {
   @Value("${storage.url}")
   private String endpoint;
 
-  @Value("${kf.download.partSize}")
-  private int partSize;
```
Comment (leoraba) [inline score-client/src/main/java/bio/overture/score/client/config/StorageConfig.java:22]: Do not remove this instance variable. as it is required on the `partCalculator` method

**Review by leoraba — CHANGES_REQUESTED** [review]
Just left a comment about the impact of removing a variable

File: score-client/src/main/java/bio/overture/score/client/config/StorageConfig.java (hunk 1 above)
Comment (Azher2Ali) [reply score-client/src/main/java/bio/overture/score/client/config/StorageConfig.java:22]: As we have removed kf profile from the applicaition.yml,` partsize `was part of kf profile which was causing NullPointerException. So we have added a default value to the partsiz.

File: score-client/src/main/resources/application.yml [hunk 2]
```diff
@@ -39,12 +39,14 @@ client:
 kfportal:
   url: https://arranger.kids-first.io/june_13/graphql
 
+download:
+  partSize: 250000000
```
Comment (joneubank) [inline score-client/src/main/resources/application.yml]: How did we decide on this changed part size from before?

Comment (Azher2Ali) [reply score-client/src/main/resources/application.yml]: We misunderstood it to different partsize, now it's been reverted to it's original part size.

**Pull request closed without merging**